src/data-workspace/inputs/validators.js

```diff
@@ -81,6 +81,12 @@ export const validatorsByValueType = {
     [VALUE_TYPES.URL]: url,
 }
 
+// minimum and maximum values are restricted to integers
+export const validatorsByValueTypeMinMax = {
+    ...validatorsByValueType,
+    [VALUE_TYPES.NUMBER]: integer,
+}
```

Comment:
I admit I was a little confused why all the other validators are in this object if limits can only be integers, but I realized it's because the existing `integer`, `integerPositive`, `integerNegative` etc. validators are functional without changing -- I have a few suggestions for clarity for [ignorant] people like me 😁 feel free to take or leave them as you like:

1. Include in this object only the value types that can have limits
2. Add to the comment above: `// (not all values can have min/max limits though)`
3. Maybe call the object `minMaxValidatorsByValueType`?

(And as mentioned in DMs, `percentage` seems to need a special min-max validator here)

Reply:
I agree with @KaiVandivier's comment above. Especially this one:
> 1. Include in this object only the value types that can have limits

And as an additional point, add a comment above the `VALUE_TYPES.NUMBER` property to emphasise the fact that this is treated as an integer, which is different than how it is treated in `validatorsByValueType`.

Reply:
Thanks @KaiVandivier and @HendrikThePendric! I've incorporated the comments above. @KaiVandivier also followed up to point out that I should restrict the PERCENTAGE to integers in 0-100 range.

When doing that, I also noticed that the logic we were using on the Inputs for these numbers was resulting in 0 values not being displayed (i.e. `format: (value) => (value ? value.toString() : '')` made 0 display as `''`; I've updated this `format: (value) => (value !== undefined ? value.toString() : '')` to avoid that issue. (I thought that should probably be fixed in the context of this PR)